src/dds-backend/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+target_link_libraries(${LRS_TARGET} PRIVATE dds)
+if(BUILD_DDS_BACKEND)
```

Comment:
We really need this here? We check it outside, if we get here we want it, no?

Reply:
Correct, will remove